gnomad_qc/v4/resources/release.py

```diff
@@ -171,26 +198,50 @@ def release_coverage_path(
         else:
             return path
     else:
-        return f"gs://gnomad/release/{release_version}/ht/{data_type}/gnomad.{data_type}.v{release_version}.coverage.ht"
+        return (
+            f"{_release_root(release_version, test=test, data_type=data_type)}/gnomad.{data_type}.v{release_version}.coverage.ht"
+        )
+
+
+def coverage_tsv_path(
```

Comment:
this might get confused with [gnomad/resources/grch38/gnomad.py](https://github.com/broadinstitute/gnomad_methods/blob/77d2be9a75fd74c0fbffafa7086fb4de5a688203/gnomad/resources/grch38/gnomad.py#L361)

Reply:
thoughts on a name change?